src/site/content/en/blog/samesite-cookies-explained/index.md

```diff
@@ -332,32 +339,125 @@ Set-Cookie: widget_session=abc123; SameSite=None; Secure
+
+Both of these changes are backwards-compatible with browsers that have correctly
+implemented earlier versions of the `SameSite` attribute, or just do not support
+it at all. By applying these changes to your cookies, you are making their
+intended use explicit rather than relying on the default behavior of the
+browser. Likewise, any clients that do not recognize `SameSite=None` as of yet
+should ignore it and carry on as if the attribute was not set.
+
+{% Aside 'warning' %} At the time of writing, the network library on iOS and Mac
+incorrectly handles unknown `SameSite` values and will treat **any unknown
+value** (including `None`) as if it was `SameSite=Strict`, which affects Safari
+on Mac and browsers wrapping WebKit on iOS (Safari, Chrome, Firefox, and
+others). This will be fixed as of macOS 10.15 and iOS 13 and is available in the
+Tech Preview now. You can track their progress on
+[WebKit Bugzilla #198181](https://bugs.webkit.org/show_bug.cgi?id=198181).
 {% endAside %}
 
-## Designing for `SameSite` cookies
-
-A core practice in software design is the
-[Separation of Concerns](https://en.wikipedia.org/wiki/Separation_of_concerns)
-which essentially states that each part of your system should have one clear
-purpose. This is more simply put in the
-[Unix philosophy](https://en.wikipedia.org/wiki/Unix_philosophy) which says,
-"Make each program do one thing well". By adding the `SameSite` attribute to
-your cookies you are creating the distinction between first-party and
-third-party usage, but a third-party cookie can still be used in a first-party
-context. It might seem simpler to just have the single cookie, but now you have
-one component doing two jobs.
-
-If you have one cookie that's providing functionality in both contexts, for
-example perhaps providing a user identifier for your main site and for an
-embeddable widget, then consider separating these into separate cookies. As
-browsers and users start to change how they accept and manage cookies providing
-this distinction makes your site more robust in scenarios where third-party
-cookies are blocked or cleared.
-
-## What should I do to enable `SameSite` today?
+### Handling incompatible clients
+
+If you have a significant number of users still using browsers or clients that
+are incompatible with these changes there are some choices for working around
+this. The general rule is to treat the incompatible clients as the special case,
+don't create an exception for browsers implementing the newer rules.
+
+At the point of sending the `Set-Cookie` header, you can choose to detect the
+client via the user agent string. For example, this snippet shows detecting iOS
+12 or Safari on Mac OS X 10.14 and serving a cookie without the `SameSite`
+attribute to those browsers. This makes use of the
+[ua-parser-js](https://www.npmjs.com/package/ua-parser-js) library for Node.js,
+it's advisable to find a library to handle user agent detection as you most
```

Comment:
I'm terrible at grammar but this seems like a run on sentence?

Reply:
You are correct. It is a specific type of run on called a comma splice. "It's" should be the start of a new sentence.

Reply:
Unspliced my comma with latest push.